### PR TITLE
Update migrate_8_13.asciidoc

### PR DESCRIPTION
**Pull request by dbrimley**
Add Notable Changes rather than Breaking Changes

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
Yes
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: docs/reference/migration/migrate_8_13.asciidoc [hunk 1]
```diff
@@ -16,14 +16,15 @@ coming::[8.13.0]
 [[breaking-changes-8.13]]
 === Breaking changes
 
-The following changes in {es} 8.13 might affect your applications
-and prevent them from operating normally.
-Before upgrading to 8.13, review these changes and take the described steps
-to mitigate the impact.
+There are no breaking changes in 8.13
```
Comment (leemthompo) [inline docs/reference/migration/migrate_8_13.asciidoc:19]:
```suggestion
There are no breaking changes in 8.13.
```

File: docs/reference/migration/migrate_8_13.asciidoc [hunk 2]
```diff
@@ -16,14 +16,15 @@ coming::[8.13.0]
...
 
+=== Notable changes
+There are notable changes in 8.13 that you need to be aware of but that we do not consider breaking, items that we may consider as notable changes are
```
Comment (leemthompo) [inline docs/reference/migration/migrate_8_13.asciidoc:22]:
```suggestion
The following are notable, non-breaking updates to be aware of:
```

File: docs/reference/migration/migrate_8_13.asciidoc [hunk 3]
```diff
@@ -16,14 +16,15 @@ coming::[8.13.0]
 [[breaking-changes-8.13]]
 === Breaking changes
 
-The following changes in {es} 8.13 might affect your applications
-and prevent them from operating normally.
-Before upgrading to 8.13, review these changes and take the described steps
-to mitigate the impact.
+There are no breaking changes in 8.13
 
+=== Notable changes
```
Comment (leemthompo) [inline docs/reference/migration/migrate_8_13.asciidoc:21]:
```suggestion
[discrete]
[[migrate-notable-changes-8.13]]
=== Notable changes
```

**Review by leemthompo — COMMENTED** [review]
Approving with some minor changes!

**Review by brianseeders — APPROVED** [review]

**Pull request merged**